docs/madr/decisions/080-inbound-policies-matches.md

```diff
@@ -170,7 +194,192 @@ policies:
+   while still allowing Service Owners to explicitly opt out by applying deny policies. (2.12)
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-mesh-operator
+spec:
+   default:
+      allow:
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://trust-domain.mesh/ns/observability"
+```
+
+4. I want to allow all clients in the `observability` namespace to access the `/metrics` endpoint on all services,
+   so that monitoring tools can collect metrics without requiring each Service Owner to configure access individually.
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-mesh-operator
+spec:
+   default:
+      allow:
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://trust-domain.mesh/ns/observability"
+           path:
+             type: Prefix
+             value: "/metrics"
+```
+
+###### Service Owner
+
+1. I want to grant access to my service to any client I choose,
+   so that I can support integrations and collaboration with other teams,
+   unless the Mesh Operator has explicitly denied that client,
+   ensuring I remain in control while respecting mesh-wide security boundaries. (2.12)
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-backend-owner
+spec:
+   targetRef:
+      kind: Dataplane
+      labels:
+         app: backend
+   default:
+      allow:
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://trust-domain.mesh/"
+```
+
+2. I want to opt out of mesh-wide `observability` access,
+   by denying requests from the `observability` namespace,
+   so that my service’s sensitive endpoints remain private unless explicitly allowed. (2.12)
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-backend-owner
+spec:
+   targetRef:
+      kind: Dataplane
+      labels:
+         app: backend
+   default:
+      deny:
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://trust-domain.mesh/ns/observability"
+```
+
+3. I want to block malicious/abusive client even if previously it was allowed,
+   so I can prevent my service from overloading until client's service team reacts to the incident. (2.12)
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-backend-owner
+spec:
+   targetRef:
+      kind: Dataplane
+      labels:
+         app: backend
+   default:
+      deny:
+         - spiffeId:
+              type: Exact
+              value: "spiffe://trust-domain.mesh/ns/default/sa/malicious"
+```
+
+4. I want to allow `GET` requests to my service from any client, but restrict `POST` requests to a group of identities,
+   so that read operations are public but write operations are gated.
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-backend-owner
+spec:
+   targetRef:
+      kind: Dataplane
+      labels:
+         app: backend
+   default:
+      allow:
+         - method: GET
+      deny:
```

Comment:
is this right? If everything is deny by default you just added a more detailed deny list 🤔

Reply:
It's not right, I should've added these POST matches to `allow`, let me fix the example